.docker/Dockerfile-template.ckan

```diff
@@ -10,9 +14,20 @@ ENV PYTHON={PYTHON}
 
 WORKDIR "${APP_DIR}"
 
-ENV DOCKERIZE_VERSION v0.6.1
-RUN apk add --no-cache build-base \
-    && curl -sL https://github.com/jwilder/dockerize/releases/download/${DOCKERIZE_VERSION}/dockerize-alpine-linux-amd64-${DOCKERIZE_VERSION}.tar.gz \
+COPY .docker/test-$VARS_TYPE.ini $CKAN_INI
+
+COPY . "${APP_DIR}"/
```

Comment:
When dyou do things in multiple steps, it creates multiple layers. i.e. if you install and remove the cache in one step, it does not save a layer with the cache.